ntbs-service/Models/Entities/DataQualityTreatmentOutcome12.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ntbs_service.Helpers;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Entities
+{
+    public class DataQualityTreatmentOutcome12 : Alert
+    {
+        public override string Action => 
+            "No treatment outcome at 12 months can be found, please provide treatment outcome with appropriate date.";
+
+        public override string ActionLink =>
+            RouteHelper.GetNotificationOverviewPathWithSectionAnchor(
+                NotificationId.GetValueOrDefault(),
```

Comment:
`NotificationId` is mandatory for this type, so would `NotificationId.Value` make more sense?